packages/cli/src/AbstractServer.ts

```diff
@@ -56,7 +59,9 @@ export abstract class AbstractServer {
 
 	protected testWebhooksEnabled = false;
 
-	constructor() {
+	readonly serverId: string;
```

Comment:
for consistency, we should either use a different name for `main` and `webhook`, or we should update `worker.ts` to also use `serverId`.